user/maven/template.py

```diff
@@ -0,0 +1,72 @@
+pkgname = "maven"
+pkgver = "3.9.9"
+pkgrel = 0
+hostmakedepends = ["openjdk21"]
+# FIXME: depends on unversioned providers are somewhat broken in apk
+# depends = ["virtual:java-jre!openjdk21-jre"]
```

Comment:
they should work fine now, as long as one of the providers has a correctly set priority